Resources/Prototypes/Species/reptilian.yml

```
@@ -26,10 +26,10 @@
     RArm: MobReptilianRArm
     LHand: MobReptilianLHand
     RHand: MobReptilianRHand
-    LLeg: MobReptilianLLeg
-    RLeg: MobReptilianRLeg
-    LFoot: MobReptilianLFoot
-    RFoot: MobReptilianRFoot
+    LLeg: MobReptilianLLegDigi # WWDP-Edit
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)